bin/compute-classpath.sh

```diff
@@ -17,36 +17,35 @@
 # limitations under the License.
 #
 
-SCALA_VERSION=2.10
-
 # Figure out where PredictionIO is installed
 FWDIR="$(cd `dirname $0`/..; pwd)"
 
 . ${FWDIR}/bin/load-pio-env.sh
 
-# Build up classpath
-CLASSPATH="${PIO_CONF_DIR}"
-
-CLASSPATH="$CLASSPATH:${FWDIR}/plugins/*:${FWDIR}/lib/spark/*"
-
-ASSEMBLY_DIR="${FWDIR}/assembly"
-
 if [ -n "$JAVA_HOME" ]; then
   JAR_CMD="$JAVA_HOME/bin/jar"
 else
   JAR_CMD="jar"
 fi
 
 # Use pio-assembly JAR from either RELEASE or assembly directory
+ASSEMBLY_DIR="${FWDIR}/assembly"
```

Comment:
I think we have a bug here and in our testing Docker builds. If you try this on a completely clean code tree, do `./make-distribution.sh` and directly run `bin/pio`, it stops working.